snowshu/adapters/snowflake_common.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Optional
+from urllib.parse import quote
+
+import sqlalchemy
+from sqlalchemy.pool import NullPool
+
+
+logger = logging.getLogger(__name__)
+
+
+class SnowflakeCommon():
+
+    def get_connection(
+        self,
+        database_override: Optional[str] = None,
+        schema_override: Optional[str] = None,
+    ) -> sqlalchemy.engine.base.Engine:
+        """Creates a connection engine without transactions.
+
+        By default, uses the instance credentials unless database or
+        schema override are provided.
+        """
+        if not self._credentials:
+            raise KeyError(
+                "Adapter.get_connection called before setting Adapter.credentials"
+            )
+
+        logger.debug(f"Acquiring {self.CLASSNAME} connection...")
```

Comment:
Is the `self.CLASSNAME` existing attribute? Shouldn't it be something like `self.__class__.__name__` ?

Reply:
I see it was inherited from BaseSqlAdapter [here](https://github.com/Health-Union/snowshu/blob/91a08038beef2298193d02195f7fdd1473c211ba/snowshu/adapters/base_sql_adapter.py#L43) earlier, but this class doesn't inherit it, so looks like this should be changed

Reply:
Good catch, I copied it but didn't notice that